ui/pages/NewCustomNetworkRequest.tsx

```diff
@@ -0,0 +1,256 @@
+    dispatch(addNetworkUserResponse([requestId, success])).then(() =>
+      window.close()
+    )
+  }
+
+  return (
+    <div className="standard_width">
+      <form
+        action="#"
+        onSubmit={(e) => {
+          e.preventDefault()
+          handleUserResponse(true)
+        }}
+      >
+        <header>
+          <h1>
+            <img width={24} alt={siteTitle} src={favicon} /> {siteTitle}
+          </h1>
+          <p>{t("addNewChain.subtitle")}</p>
+        </header>
+        <div className="details_container">
+          <div className="add_chain_imgs">
+            <div className="new_chain_logo_wrapper">
+              {iconUrl ? (
+                <div className="new_chain_logo" />
+              ) : (
+                <SharedNetworkIcon
+                  size={50}
+                  network={{
+                    name: chainName,
+                    chainID,
+                    family: "EVM",
+                    baseAsset: { ...nativeCurrency, chainID },
+                  }}
+                />
+              )}
+            </div>
+            <div className="plus_wrapper">
+              <SharedIcon
+                width={16}
+                height={16}
+                color="var(--trophy-gold)"
+                icon="plus@2x.png"
+              />
+            </div>
+            <div className="tally_logo" />
+          </div>
+          <dl className="chain_details">
+            <div className="row">
+              <dt>{t("addNewChain.name")}</dt>
+              <dd>{chainName}</dd>
+            </div>
+            <div className="row">
+              <dt>{t("addNewChain.chainId")}</dt>
+              <dd>{chainID}</dd>
+            </div>
+            <div className="row">
+              <dt>{t("addNewChain.currency")}</dt>
+              <dd>{nativeCurrency.symbol}</dd>
+            </div>
+            <dt>{t("addNewChain.rpc")}</dt>
+            <dd className="rpc_url">{rpcUrls[0]}</dd>
+            <dt>{t("addNewChain.explorer")}</dt>
+            <dd>{blockExplorerUrl}</dd>
+          </dl>
+        </div>
+        <footer>
+          <SharedButton
+            size="large"
+            type="secondary"
+            onClick={() => handleUserResponse(false)}
+          >
+            {t("addNewChain.cancel")}
+          </SharedButton>
+          <SharedButton size="large" type="primary" isFormSubmit>
+            {t("addNewChain.submit")}
+          </SharedButton>
+        </footer>
+      </form>
+      <style jsx>{`
+        .add_chain_imgs {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          padding-bottom: 24px;
+          border-bottom: 1px solid var(--castle-black);
+        }
+
+        .new_chain_logo_wrapper {
+          width: 50px;
+          height: 50px;
+          border-radius: 8px;
+          overflow: hidden;
+        }
+
+        .new_chain_logo {
+          width: 50px;
+          height: 50px;
```

Comment:
These icons should have the same size: 
<img width="211" alt="image" src="https://user-images.githubusercontent.com/20949277/220135473-68a6e94a-f968-4c19-bccd-289eb160099b.png">

```suggestion
          width: 56px;
          height: 56px;
```